R/launcher-functions.R

```diff
@@ -0,0 +1,326 @@
+#' Retrieve Launcher Information
+#'
+#' Retrieve information about the launcher, as well as the different clusters
+#' that the launcher has been configured to use.
+#'
+#' @name launcher
+#' @export
+launcherGetInfo <- function() {
+  callLauncherFun("launcher.getInfo")
+}
+
+#' Check if Launcher is Available
+#'
+#' Check if the RStudio launcher is currently available.
```

Comment:
More specifically, this checks to see if the launcher is configured to support running ad-hoc launcher jobs. It is possible to have the launcher configured for launcher-based sessions but have ad-hoc launcher jobs disabled (in which case this API returns FALSE). Clear as mud, I know. Not sure this is worth trying to explain (or that it will cause any issues) so feel free to leave this as is!